backoffice/backoffice/users/views.py

```diff
@@ -43,3 +45,13 @@ def get_redirect_url(self):
 
 
 user_redirect_view = UserRedirectView.as_view()
+
+
+def user_login_success(request):
+    refresh = RefreshToken.for_user(request.user)
```

Comment:
wouldn't this fail if someone goes directly to `login/success` and there is no user?